scripts/kube-hunter.sh

```diff
@@ -1,3 +1,19 @@
 #!/bin/bash
 
-exit 0
+set -euo pipefail
+
+readonly dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Install the kube-hunter job
+kubectl apply -f "${dir}/kube-hunter-manifests/config.yaml"
+pod_name=$(kubectl get pod -o name -l job-name=kube-hunter)
+
+while [ "$(kubectl get "${pod_name}" -o jsonpath='{.status.phase}')" != "Succeeded" ]; do
+  echo "Waiting until the 'kube-hunter' job is complete..."
+  sleep 1
+done
+
+echo "--------------------------------------------------"
+echo "kube-hunter output:"
+echo "--------------------------------------------------"
+kubectl logs "${pod_name}"
```

Comment:
Should we somehow check the status of the kube-hunter here and exit with error if it found something?

Reply:
Currently the part of the story/work is to just post the output. And as a followup work is to create the error reporting mechanism.